source/components/organisms/Step/StepFooter/StepFooter.tsx

```diff
@@ -83,7 +83,15 @@ const StepFooter: React.FC<Props> = ({
         return onSubmit || null;
       }
       case 'backToMain': {
-        return formNavigation.goToMainForm;
+        return () => {
+          const errorCallback = () => {};
+  
+          const onValidCallback = async () => {
```

Comment:
Why does this function has to be async when we aren't awaiting anything?

Reply:
Great question, fixed!